runtime/doc/vim9class.txt

```diff
@@ -312,14 +321,38 @@ as the first character in the method name: >
 	    echo "Foo"
 	enddef
 	def Bar()
-	    OtherThing._Foo()
+	    _Foo()
 	enddef
     endclass
 <
 							*E1370*
 Note that constructors cannot be declared as "static", because they always
 are.
 
+To access the class methods and class variables of a super class in an
+extended class, the class name prefix should be used: >
```

Comment:
Is it worth mentioning that this is how they are access from anywhere other than the defining class. Maybe something like
```
... class name prefix should be used, _just as from anywhere outside of the defining class_: >

Reply:
Yes. I will update the text.  Let me know if the vim9class.txt file needs to be updated with any additional information to reflect the recent changes in the specification.